tests/static/validate-client-tests.html

```diff
@@ -0,0 +1,170 @@
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/Fluid.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/FluidPromises.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/FluidDOMUtilities.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/FluidIoC.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/DataBinding.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/FluidView.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/lib/fastXmlPull/js/fastXmlPull.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/renderer/js/fluidParser.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/renderer/js/fluidRenderer.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/FluidRequests.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/ModelTransformation.js"></script>
+        <script type="text/javascript" src="/modules/infusion/src/framework/core/js/ModelTransformationTransforms.js"></script>
+
+
+        <!-- Client-side parser dependencies -->
+        <script type="text/javascript" src="/modules/json-schema-ref-parser/dist/ref-parser.js"></script>
+        <script src="/src/js/common/parser.js"></script>
+
+        <!-- Client-side validator dependencies -->
+        <!-- browserified AJV dependency -->
+        <script type="text/javascript" src="/content/js/ajv.bundle.js"></script>
+        <script src="/src/js/common/validate.js"></script>
+    </head>
+    <body>
+        <p>This page instantiates a client-side validator, which we will exercise using Zombie.  This page must be hosted using gpii.express or another web server.</p>
+
+        <script type="text/javascript">
+            var clientValidator = gpii.schema.validator({
+                model: {
+                    schemas: {
+                        "base.json": {
+                            "$schema": "http://json-schema.org/draft-04/schema#",
+                            "id":      "base.json#",
+                            "title":   "Example Schema",
+                            "type":    "object",
+                            "definitions": {
+                                "required": {
+                                    "type": "boolean"
+                                },
+                                "boolean": {
+                                    "type": "boolean"
+                                },
+                                "date": {
+                                    "type":   "string",
```

Comment:
Spacing has become rather peculiar in these multi-line JSON values. Also, can we break out this giant script into a js include? That way it can be linted properly